### PR TITLE
add login page, upgrade home page style

### PR DESCRIPTION
**Pull request by leoendless**

### CODE REVIEW

File: src/components/Header/index.jsx [hunk 1]
```diff
@@ -1,27 +1,30 @@
...
     isHome: PropTypes.bool,
   }
 
   render() {
-    const { isHome } = this.props;
-
-    const logoUrl = isHome ? '../../assets/logo.svg' : '../../assets/logo2.svg';
+    const { isHome, rootStore: { fixNav } } = this.props;
+    const isDark = !isHome || fixNav;
+    const logoUrl = isDark ? '../../assets/logo2.svg' : '../../assets/logo.svg';
```
Comment (libangrui) [inline src/components/Header/index.jsx]: ``../../assets/${isDark ? "logo2" : "logo"}.svg`` 
Is it better?

File: src/containers/Home/index.jsx [hunk 2]
```diff
@@ -1,27 +1,44 @@
...
+  componentDidMount() {
+    window.onscroll = this.handleScoll;
+  }
+
+  handleScoll = () => {
+    const store = this.props.rootStore;
+    if (!store.fixNav && getScrollTop() > 345) {
+      store.fixNav = true;
+    } else if (store.fixNav && getScrollTop() < 345) {
+      store.fixNav = false;
```
Comment (libangrui) [inline src/containers/Home/index.jsx]: `props` can be modified ?

File: src/components/Banner/index.jsx [hunk 3]
```diff
@@ -8,10 +8,13 @@ export default class Footer extends PureComponent {
     return (
       <div className={styles.banner}>
         <div className={styles.wrapper}>
+          <img className="banner-img-1" src="../../assets/1-1.svg" alt=""/>
+          <img className="banner-img-2" src="../../assets/1-2.svg" alt=""/>
+          <img className="banner-img-3" src="../../assets/1-3.svg" alt=""/>
```
Comment (libangrui) [inline src/components/Banner/index.jsx]: The pre-path `../../assets/`  may be abstracted.

File: src/scss/variables.scss [hunk 4]
```diff
@@ -16,7 +16,7 @@ $grey-light: #d6cfde;
 
 $id-color: #127db3;
 $link-color: #6c2db5;
-$title-color: #2f0d56;
+$title-color: #300E56;
```
Comment (libangrui) [inline src/scss/variables.scss]: `title-color` has a wide range of effects. `#2f0d56` is used in many places.
Has the designer been able to confirm this change?

File: src/containers/Home/index.jsx (hunk 2 above)
Comment (leoendless) [reply src/containers/Home/index.jsx]: em, i'd better to use mobx action to modify state

File: src/components/Header/index.jsx (hunk 1 above)
Comment (leoendless) [reply src/components/Header/index.jsx]: seems no benefit, and string template will generate new string every time

File: src/scss/variables.scss (hunk 4 above)
Comment (leoendless) [reply src/scss/variables.scss]: i will check it

**Pull request merged**